Odata-docs/odatalib/v7/simplified-options.md

```diff
@@ -1,8 +1,8 @@
 ---
 title: "Centralized control for OData Simplified Options"
-description: ""
+description: "From ODataV7.0, we add a centralized control class for them, which is ODataSimplifiedOptions."
```

Comment:
```suggestion
description: "This section describes how to use ODataSimplifiedOptions."
```

Reply:
revised and replaced with the suggested description.